app/furniture/marketplace/product_component.rb

```diff
@@ -0,0 +1,48 @@
+class Marketplace
+  class ProductComponent < ApplicationComponent
+    attr_accessor :product
+    delegate :name, :description, :location, to: :product
+
+    def initialize(product:, data: {}, classes: "", **kwargs)
+      super(data: data, classes: classes, **kwargs)
+
+      self.product = product
+    end
+
+    def edit_button
+      return unless edit_button?
+
+      Buttons::SecondaryComponent.new label: "#{t("icons.edit")} #{t("edit.link_to")}",
+        title: t("marketplace.products.edit.link_to", name: name),
+        href: location(:edit), turbo_stream: true,
+        method: :get
+    end
+
+    def tax_rates
+      product.tax_rates.map do |tax_rate|
+        "#{tax_rate.label} #{helpers.number_to_percentage(tax_rate.tax_rate, precision: 2)}"
+      end.to_sentence
+    end
+
+    def price
+      helpers.humanized_money_with_symbol(product.price)
+    end
+
+    def edit_button?
+      product.persisted? && policy(product).edit?
+    end
+
+    def destroy_button
+      return unless destroy_button?
+
+      Buttons::SecondaryComponent.new label: "#{t("icons.destroy")} #{t("destroy.link_to")}",
+        title: t("marketplace.products.destroy.link_to", name: product.name),
+        href: product.location, turbo_stream: true,
+        method: :delete
```

Comment:
Should `confirm: t("destroy.confirm")` be added here like https://github.com/zinc-collective/convene/pull/1357 did until we build the [archive products feature](https://github.com/zinc-collective/convene/pull/1357#pullrequestreview-1382646211)?